crates/bevy_render/src/render_resource/buffer_vec.rs

```diff
@@ -417,25 +424,30 @@ where
     /// Queues writing of data from system RAM to VRAM using the [`RenderDevice`]
     /// and the provided [`RenderQueue`].
     ///
-    /// Before queuing the write, a [`reserve`](BufferVec::reserve) operation
-    /// is executed.
+    /// If the buffer is not initialized on the gpu or the range is bigger than the capacity it will
+    /// return an error. You'll need to either reserve a new buffer which will lose data on the gpu
```

Comment:
```suggestion
    /// If the buffer is not initialized on the GPU or the range is bigger than the capacity it will
    /// return an error. You'll need to either reserve a new buffer which will lose data on the GPU
```